sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -551,6 +554,7 @@ export interface TopicProperties {
     enableBatchedOperations: boolean;
     readonly enableExpress: boolean;
     readonly enablePartitioning: boolean;
+    maxMessageSizeInKilobytes?: number;
```

Comment:
If this is always populated (as indicated in one of the comments), should it be nullable?

Reply:
Good point, will update